src/actions/userArticles.action.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+import config from '../config';
+import { GET_USER_ARTICLE_SUCCESS, GET_USER_ARTICLE_FAILURE } from './types';
+
+const userArticles = username => dispatch => axios.get(`${config.apiUrl}/articles?user=${username}`)
+  .then((result) => {
+    dispatch({
+      type: GET_USER_ARTICLE_SUCCESS,
```

Comment:
Good Job Kingsley! But why do not have a separate function for your Action types outside the main Action? Doing it that way makes the code more readable.